package.json

```diff
@@ -88,6 +88,11 @@
                     ],
                     "default": "off",
                     "description": "Traces the communication between VSCode and the YML language server."
+                },
+                "yseopml.kaoFiles": {
+                    "scope": "window",
+                    "type": "array",
+                    "description": "List of the project.kao files that should parsed when initializing the workspace."
```

Comment:
What would happen if we just did something ⇔ `find my-workspace/ -type f -name project.kao`? xD
Oh but you actually have a `glob('**/project.kao', …)` when this is not set. So I guess we kinda do that. Edit: oh but only to fetch _one_ file. :thinking: